enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/core/consensus/roles/PreVotingVoterTest.java

```diff
@@ -52,7 +52,8 @@ public void shouldRespondPositivelyIfWouldVoteForCandidate() throws Exception
         RaftState raftState = initialState();
 
         // when
-        Outcome outcome = new Follower().handle( new RaftMessages.PreVote.Request( member1, 0, member1, 0, 0 ), raftState, log() );
+        Outcome outcome = new Follower().handle( new RaftMessages.PreVote.Request( member1, 0, member1, 0, 0 ),
```

Comment:
This file seems unnecessarily reformatted.

Reply:
Well spotted. This would have been a left over from the abandoned approach of including an Instant in all RaftMessages.